cmd/peek-submission/main_test.go

```diff
@@ -22,30 +22,38 @@ func TestPeekBase(test *testing.T) {
 		targetSubmission    string
 		expectedSubmimssion string
 		expectedExitCode    int
+		expectedLocator     string
+		logLevel            string
 	}{
-		{"course-student@test.edulinq.org", "course101", "hw0", "", "1697406272", 0},
-		{"course-student@test.edulinq.org", "course101", "hw0", "1697406272", "1697406272", 0},
-		{"course-student@test.edulinq.org", "course101", "hw0", "course101::hw0::student@test.com::1697406256", "1697406256", 0},
+		{"course-student@test.edulinq.org", "course101", "hw0", "", "1697406272", 0, "", ""},
+		{"course-student@test.edulinq.org", "course101", "hw0", "1697406272", "1697406272", 0, "", ""},
+		{"course-student@test.edulinq.org", "course101", "hw0", "course101::hw0::student@test.com::1697406256", "1697406256", 0, "", ""},
 
-		{"course-admin@test.edulinq.org", "course101", "hw0", "", "", 0},
-		{"course-student@test.edulinq.org", "course101", "hw0", "ZZZ", "", 0},
+		{"course-admin@test.edulinq.org", "course101", "hw0", "", "", 0, "", ""},
+		{"course-student@test.edulinq.org", "course101", "hw0", "ZZZ", "", 0, "", ""},
 
-		{"course-student@test.edulinq.org", "ZZZ", "hw0", "", "", 2},
-		{"course-student@test.edulinq.org", "course101", "ZZZ", "", "", 2},
-		{"course-student@test.edulinq.org", "ZZZ", "ZZZ", "", "", 2},
+		{"course-student@test.edulinq.org", "ZZZ", "hw0", "", "", 2, "-018", "error"},
+		{"course-student@test.edulinq.org", "course101", "ZZZ", "", "", 2, "-022", "error"},
+		{"course-student@test.edulinq.org", "ZZZ", "ZZZ", "", "", 2, "-018", "error"},
 	}
 
+	util.ShouldExit = false
+	defer func() {
+		util.ShouldExit = true
+	}()
+
+	oldExitCode := util.GetLastExitCode()
+	defer func() {
+		util.SetExitCode(oldExitCode)
+	}()
```

Comment:
No need to track/replace the last exit code.
We just need it for the test cases.